### ISSUE TITLE
Captain Morgan for Kitty

### ISSUE THREAD

**Issue by JSpringett** [abersailbot/kitty-hardware#6]

**Comment by ghost** [comment]
If the bodged Canada board (w/ ribbon cable) works the same, easy peasy.

If not (for now, supposedly), just edit the software to cope.

**Comment by Danyc0** [comment]
Please don't use the bodged canada board for Kitty. Spin a new one! The canada board is not very well protected in terms of grounding and reverse polarity protection. Also it's much harder to use for debugging because it's missing some of the more useful LEDs

**Comment by JSpringett** [comment]
Its only temporary will we wait for the new board to be sorted out.

**Comment by naturesyouth** [comment]
i think its a good idea to use the old shield to get both boats as similar as possible as soon as possible, but given the large number of unknowns in kitty i would feel better if you transplanted the board from dewi into kitty as it is so much safer.

**Comment by Danyc0** [comment]
It's also worth considering though, that Kitty has a lot more room in her boxes to fit the old ribbon-cable/stacked version of Captain-Morgan

**Comment by naturesyouth** [comment]
yeah but i would feel better trusting kitty to the better assembled 1.2, blowing a pi/battery/anything else would be less than fun, especially as it will be replaced by christmas and we are currently rediscovering how kitty works.
